Model/DAO/SkillDAO.py

```diff
@@ -7,13 +7,30 @@ class SkillDAO:
     def __init__(self):
         pass
 
-    def get_skill(self, id):
+    def get_matching_images(self, requested_attrs):
```

Comment:
I think we are using camelCase. We can discuss changing this since I know python typically uses snake case